src/linker/Linker/MessageSubcategory.cs

```diff
@@ -7,6 +7,7 @@ namespace Mono.Linker
 	public static class MessageSubCategory
 	{
 		public const string None = "";
+		public const string TrimCorrectness = "Trim correctness";
```

Comment:
What is this for? I don't think this term is broadly understandable

Reply:
I agree it might not be very informative to all users, but its main purpose is to group all warnings related to trimming analysis. Going forward, if we ever allow subcategory mapping in `nowarn` (things like `--nowarn trimcorrectness` = `--nowarn IL2006;IL2026;IL2043`) it might come handy. What do you think could be a more informative/useful subcategory name/message?

Reply:
I think something like "trimalert" for the flag and "Trim alert" for subcategory printing is more readable.
`--nowarn trimalert` explains the purpose
`Trim alert warning IL2006 : Mono.Linker.Tests....` 
No warn on correctness sounds like the program warns when is correct and we are disabling it. Also, since it's a broader term sounds like it should have a lot more warnings inside the subcategory.

Reply:
Given the discussion on the public property - maybe "TrimAnalysis" ?

Reply:
I think `TrimAnalysis` is a good option given that we'll probably have an `EnableTrimAnlaysis` property in the SDK. `Correctness` is even broader than `TrimCorrectness`.